### PR TITLE
Node IP for Internal and External Clusters

### PR DESCRIPTION
**Pull request by oumkale**
## Proposed changes
This PR has Screens and Modals for targets.

## Types of changes

What types of changes does your code introduce to Litmus?
- [x] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

File: litmus-portal/graphql-server/pkg/cluster/util.go [hunk 1]
```diff
@@ -20,3 +29,57 @@ func VerifyCluster(identity model.ClusterIdentity) (*database.Cluster, error) {
...
+			panic(err.Error())
+		}
+	}
+
+	// creates the clientset
+	clientset, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		panic(err.Error())
+	}
+	nodeName := os.Getenv("NODE_NAME")
```
Comment (gdsoumya) [inline litmus-portal/graphql-server/pkg/cluster/util.go:65]: Where is this env variable set?

File: litmus-portal/k8s-manifest.yml [hunk 2]
```diff
@@ -69,7 +69,10 @@ spec:
               value: "litmus-portal@123"
             - name: EXTERNAL_ADDRESS
               value: ""
-            - name: SERVICE_ADDRESS
+            - name: NODE_NAME
```
Comment (oumkale) [inline litmus-portal/k8s-manifest.yml:72]: @gdsoumya I have set env variables here.

Comment (gdsoumya) [reply litmus-portal/k8s-manifest.yml:72]: It was missing before. it's good now

Comment (oumkale) [reply litmus-portal/k8s-manifest.yml:72]: Thank you @gdsoumya for your review.
Yaaa Sorry for that. That was my mistake.

**Comment by gdsoumya** [comment]
Have you tested it? Does it run on minikube? It is not working on my setup.

**Comment by oumkale** [comment]
@gdsoumya I have tested it. But it's been a long time. Please will you conform to me? What is the problem now?

**Comment by gdsoumya** [comment]
@oumkale self-deployer is not working with nodeIP

**Pull request closed without merging**